irohad/main/iroha_conf_loader.hpp

```diff
@@ -28,6 +28,24 @@ namespace config_members {
   const char *MstSupport = "mst_enable";
 }  // namespace config_members
 
+static constexpr size_t kBadJsonPrintLength = 15;
+static constexpr size_t kBadJsonPrintOffsset = 5;
+static_assert(kBadJsonPrintOffsset <= kBadJsonPrintLength,
```

Comment:
quite useless

Reply:
May be so, but i believe it does not slow compilation, prevents accidental changes from crashing runtime and the comment for this assertion helps understanding the relation between these consts.